Arithmetic/Exercise_6/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+
+namespace Exercise_6
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var number = 110;
+
+
+            for (var i = 1; i <= number; i++)
+            {
+                var coza = (i % 3 == 0) && (i % 7 != 0) && (i % 5 != 0);
```

Comment:
Šis gan jau strādā, bet izskatās ļoti neefektīvi. Tev katrs cipars iet cauri katram statementam un tad visiem if, jo tie pat nav if else. Ja vairāk izmantotu if else, kods būtu "vieglāks"